### ISSUE TITLE
ESL Challenge

### ISSUE THREAD

**Issue by gbltraveler** [MicrosoftDocs/microsoft-style-guide#66]
I'm not an English language expert, but I have spent a fair amount of time working with classrooms of learners who are using English as a second language.  I am wondering if using, "Bigger, fewer words", really works for a reader who is looking at documentation as an English as a second language consumer?  My sense is that does not, but only anecdotal evidence.  Anyway, I thought it worth mentioning.  Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 23388da6-bce7-7800-0335-38f82eedaf3c
* Version Independent ID: 7934337b-70be-7223-4e76-408f7086736a
* Content: [Top 10 tips for Microsoft style and voice - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#feedback)
* Content Source: [styleguide/top-10-tips-style-voice.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/top-10-tips-style-voice.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by LLawsonBruton** [comment]
I agree. I write for an international audience and I do everything I can to keep the words to 2 syllables or less.
